msamp/megatron/distributed.py

```diff
@@ -0,0 +1,213 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""The DistributedDataParallel which supports FP8."""
+
+import math
+import torch
```

Comment:
package `torch` is the same with megtraon, both are third party packages, while math is native package. Put torch together with megatron.

Reply:
Good point. Fixed.